workshop-registration/features/environment.py

```diff
@@ -23,3 +55,45 @@ def before_scenario(context, scenario):
                 presenter=context.list_registrants_presenter,
         )
     )
+
+
+def access_token():
+    load_dotenv()
+    oauth = OAuthGetToken(
+        client_id=os.environ["CLIENT_ID"],
+        client_secret=os.environ["CLIENT_SECRET"],
+        account_id=os.environ["ACCOUNT_ID"],
+    )
+    access_data = oauth.create_access_token()
+    return access_data["access_token"]
+
+
+
+@fixture
+def create_zoom_registrant(access_token: str, meeting_id: str, status: Literal["approved","pending","denied"]) -> str:
+    fname = 'test'+str(random.randint(1, 100))
+    params = {
+    "first_name": fname,
+    "last_name": 'last_name',
+    "email": f'eve{random.randint(1, 100)}@lname.com',
+    "custom_questions": [{'title':'Research Group', 'value': 'AG Bashiri'}],
+    "status": status
+    }
+    response = requests.post(
+        url=f"https://api.zoom.us/v2/meetings/{meeting_id.replace(' ','')}/registrants",
+        headers={"Authorization": f"Bearer {access_token}"},
+        json=params,
+        )
+    response.raise_for_status()
+    return response.json()['registrant_id']
+    
+
+@fixture
+def after_scenario(context, scenario):
+     if 'change_status_on_zoom_side' in scenario.tags:
+        registrant_id = context.registrant_id
+        response = requests.delete(
+            url=f"https://api.zoom.us/v2/meetings/{context.meeting_id.replace(' ', '')}/registrants/{registrant_id}",
+            headers={"Authorization": f"Bearer {context.access_token()}"},
+            )
+        response.raise_for_status()
```

Comment:
Similar to the `create_zoom_registrant` function these lines could also be their own function: `remove_zoom_registrant`